report/topology.go

```diff
@@ -13,7 +13,7 @@ type Topology struct {
 	Tag               string            `json:"tag,omitempty"`
 	Label             string            `json:"label,omitempty"`
 	LabelPlural       string            `json:"label_plural,omitempty"`
-	Nodes             Nodes             `json:"nodes"`
+	Nodes             Nodes             `json:"nodes,omitempty" deepequal:"nil==empty"`
```

Comment:
I want to check how `deepequal` here works but don't know what exactly to Google.. Do you have links/references at hand?

Reply:
that feature was added here: https://github.com/weaveworks/scope/commit/f44f8806cd7f20036362f73f6c749294fecc61b2#diff-1aaeb558d4801562ecccd7b7352209dbR149